src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/Interfaces/IReleaseStatusService.cs

```diff
@@ -15,6 +15,10 @@ public interface IReleaseStatusService
 
         Task<ReleaseStatus> GetAsync(Guid releaseId, Guid releaseStatusId);
 
+        Task<IEnumerable<ReleaseStatus>> GetWherePublishingDueTodayWithStages(ReleaseStatusContentStage? content = null,
+            ReleaseStatusDataStage? data = null, ReleaseStatusFilesStage? files = null,
```

Comment:
Not a big deal, but would just tidy up these arguments all onto new lines (same applies for elsewhere)

Reply:
Thanks, I've done that now.